src/tests/dao.rs

```diff
@@ -278,6 +278,84 @@ fn test_dao_single_cell() {
     verify_result.expect("pass verification");
 }
 
+#[test]
```

Comment:
Suggest adding some failing cases for WitnessArgs.

Reply:
Done